packages/python-packages/doc-warden/README.md

```diff
@@ -224,7 +224,7 @@ the `package_indexing_exclusion_list` array members to enable just this sort of
 
 `package_indexing_traversal_stops` is used during parse of .NET language repos _only_. This is due to how the discovery logic for readme and changelog is implemented for .NET projects. Specifically, readmes for a .csproj are often a couple directories up from their parent .csproj location!
 
-For .net, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
+For .NEt, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
```

Comment:
```suggestion
For .NET, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
```

Reply:
Again? Rats. Let me double check. Seems I released the `shift` key too soon.

Reply:
Via VSCode, that was the only other one I found. Thanks.